### DISCUSSION TITLE
Github keeps denying my application

### DISCUSSION THREAD

**Discussion by freek925** [community/community#145054] [GitHub Education]
### Hi everyone,

I've had the student pack for a couple of years now and I had to reverify my status. No issues I thought, I've done this like 3 times now.
However Github keeps denying my application no matter what proof I provide. 
I don't have a student pass with a year on it but my school provides an official proof of enrollment Github keeps on denying. Everything else I try just keeps on getting denied.
Why is it so goddamn hard to get reverified ever though I've provided more proof then any time before.

**Comment by pierreparre** [comment]
same here, never sent that many proofs and still got rejected many times.
getting annoying
